marklogic-data-hub-central/ui/src/components/sidebar/sidebar.tsx

```diff
@@ -44,9 +44,11 @@ const Sidebar: React.FC<Props> = (props) => {
   let integers = ['int', 'integer', 'short', 'long'];
   let decimals = ['decimal', 'double', 'float'];
   const dateRangeOptions = ['Today', 'This Week', 'This Month', 'Custom'];
+  const [activeKey, setActiveKey] = useState<any[]>([]);
 
   useEffect(() => {
     if (props.facets) {
+      props.selectedEntities.length === 0 ? setActiveKey(['entityProperties','hubProperties']) : setActiveKey(['entityProperties']);
```

Comment:
entityProperties dont show up when "All Entities" is selected. So I think it is safe to say 
      `props.selectedEntities.length === 0 ? setActiveKey(['hubProperties']) : setActiveKey(['entityProperties']);`

Reply:
I agree, Will make the change.